content/en/security/cloud_security_management/troubleshooting/threats.md

```diff
@@ -83,6 +83,83 @@ datadog:
 ```bash
 DD_RUNTIME_SECURITY_CONFIG_NETWORK_ENABLED=false
 ```
+## Disable CSM Threats
+
+To disable CSM Threats, follow the steps for your Agent platform.
+
+### Helm
+
+In the Helm `values.yaml`, set `securityAgent.runtime` to `enabled: false` as follows:
+
+```
```

Comment:
I think the code block shortcode is preferred to code fences, unless you can't use the shortcode here for some reason?

Reply:
fixed. thanks